### PR TITLE
support right to left languages 

### PR DESCRIPTION
**Pull request by steins-code**
flip images when needed and naturally align text
IMG_9E277D5655F3-1

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/InfoCollectionViewCell.xib [hunk 1]
```diff
@@ -114,13 +114,13 @@
...
-            <color red="0.090000003576278687" green="0.097999997437000275" blue="0.10199999809265137" alpha="0.10000000149011612" colorSpace="custom" customColorSpace="sRGB"/>
+            <color red="0.090196078431372548" green="0.098039215686274508" blue="0.10196078431372549" alpha="0.10000000149011612" colorSpace="custom" customColorSpace="sRGB"/>
         </namedColor>
         <namedColor name="ENA Text Primary 1 Color">
-            <color red="0.090000003576278687" green="0.097999997437000275" blue="0.10199999809265137" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+            <color red="0.090196078431372548" green="0.098039215686274508" blue="0.10196078431372549" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
         </namedColor>
         <namedColor name="ENA Text Primary 2 Color">
-            <color red="0.090000003576278687" green="0.097999997437000275" blue="0.10199999809265137" alpha="0.60000002384185791" colorSpace="custom" customColorSpace="sRGB"/>
+            <color red="0.090196078431372548" green="0.098039215686274508" blue="0.10196078431372549" alpha="0.60000002384185791" colorSpace="custom" customColorSpace="sRGB"/>
```
Comment (MichaelPFrey) [inline src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/InfoCollectionViewCell.xib:123]: are the changes in this file necessary or are they unintentional artifacts from e.g. using a GUI editor software?
_(in general, it is good practice to avoid committing unnecessary changes, as such artifacts tend to cause merge conflicts)_

**Review by haosap — APPROVED** [review]

**Comment by haosap** [comment]
I put it onhold, because we need to clarify if it's part of Sprint 2

**Comment by steins-code** [comment]
for now we close it
